Automattic/wp-calypso
# Login: Update OAuth design for mobile

This makes a few tweaks to the design of the OAuth pages for mobile, and cleans up the code, removing unused selectors.
  
#### Testing instructions
  
1. Run `git checkout update/oauth-design` and start your server
2. Check the layout of the custom OAuth pages:
http://calypso.localhost:3000/log-in?client_id=50916
http://calypso.localhost:3000/log-in?client_id=930
http://calypso.localhost:3000/log-in?client_id=973
http://calypso.localhost:3000/log-in?client_id=1854
http://calypso.localhost:3000/log-in?client_id=978
3. Also check a standard OAuth page:
http://calypso.localhost:3000/log-in?client_id=978

#### Reviews
